examples/loaders/utils.py

```diff
@@ -0,0 +1,28 @@
+"""Utility functions for loaders examples."""
+import requests
+from tqdm import tqdm
+
+
+def download(url: str, fname: str, chunk_size: int = 1024) -> None:
```

Comment:
`download_file` and `filename` would be more descriptive
```suggestion
def download_file(url: str, filename: str, chunk_size: int = 1024) -> None:
```